recipes/libnetfilter_log/all/conanfile.py

```diff
@@ -0,0 +1,67 @@
+import os
+
+from conan import ConanFile
+from conan.tools.gnu import Autotools, AutotoolsToolchain, PkgConfigDeps
+from conan.tools.files import copy, get, rmdir, rm
+from conan.tools.layout import basic_layout
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=2.4.0" # for attribute languages
+
+class Libnetfilter_logConan(ConanFile):
+    name = "libnetfilter_log"
+    license = "GPL-2.0-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://netfilter.org/projects/libnetfilter_log/index.html"
+    description = "Library providing interface to packets that have been logged by the kernel packet filter"
+    topics = ("libnfnetlink_log", "nftables")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    package_type = "library"
+    languages = ["C"]
+    implements = ["auto_shared_fpic"]
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("libmnl/1.0.4")
+        self.requires("libnetfilter_conntrack/1.0.9")
```

Comment:
I see that autotools is looking for this package when configuring, but then it's not present in either of the upstream .pc files even though it seems to be used at some points in the code. Do you have any insight into it? Thanks!